README.md

```diff
@@ -16,3 +16,19 @@ Current list of distributions:
 
 - [OpenTelemetry Collector (also known as "otelcol")](./distributions/otelcol)
 - [OpenTelemetry Collector Contrib (also known as "otelcol-contrib")](./distributions/otelcol-contrib)
+
+## Criteria for Distributions
+
+- To honor commitments made by OpenTelemetry to other Open Source projects the Collector SIG should support at least 1 distribution that includes support for Prometheus, Jaeger, Zipkin, and OpenCensus.
+- Distributions supported by the Collector SIG should serve a specific purpose and those purposes should have minimal overlap.
+- Distributions supported by the Collector SIG should meet general needs and not be too niche.
+- Distributions supported by the Collector SIG should only target the needs of the OpenTelemetry project.
```

Comment:
Do we have a place where we define the 'needs' of the OpenTelemetry project? It seems like the first one (commitments made by OpenTelemetry to other OSS) is derived from this one, should we group them somehow?

Reply:
This item came from [this discussion](https://github.com/open-telemetry/opentelemetry-collector-releases/issues/360#issuecomment-1607626029).  It was an attempt to clarify that we don't want to support distributions for vendors or other ecosystems. I am open to wordsmithing.